package calculatorr.using.function;
import java.lang.Math;
import java.util.Scanner;

public class RuntimeException extends Throwable {
//double n, n0;
   public RuntimeException(String message){
//      System.out.println("Re enter the num");
//      Scanner sr = new Scanner(System.in);
//      double value = sr.nextDouble();
//      double value2 = sr.nextDouble();
//
//      Input i = new Input(n,n0);
//      Main m = new Main();
//      Input val = m.call();

      System.out.println("Power is = "+Math.pow(Main.n1, Main.n2));

   }
}
